src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -13,6 +13,10 @@ case class ThurloeKeyValue(
   userId: Option[String] = None,
   keyValuePair: Option[FireCloudKeyValue] = None)
 
+case class ThurloeKeyValues(
```

Comment:
And just to be clear, I'd be perfectly happy if this PR went in as-is and the research (with a potential follow-up PR) came later.